packages/swing-store-lmdb/src/lmdbSwingStore.js

```diff
@@ -167,200 +166,7 @@ function makeLMDBSwingStore(dirPath, forceReset, options) {
-      // not ideal, but it works.
-      //
-      // const innerReader = new Readlines(fd, { position: startOffset });
-      const innerReader = new Readlines(fd);
-      function* reader() {
-        try {
-          while (true) {
-            assert(
-              streamStatus.get(streamName) === readStatus,
-              X`can't read stream ${q(streamName)}, it's been closed`,
-            );
-            const line = /** @type {string|false} */ (innerReader.next());
-            // N.b.: since uncommitted writes may leave an overhang of data in
-            // the stream file, the itemCount is the true indicator of the end
-            // of the stream, not the point at which the line reader reaches the
-            // end-of-file.
-            if (line && itemCount > 0) {
-              itemCount -= 1;
-              const result = line.toString();
-              if (skipCount > 0) {
-                skipCount -= 1;
-              } else {
-                yield result;
-              }
-            } else {
-              closefd(fd);
-              break;
-            }
-          }
-        } catch (e) {
-          console.log(e);
-        } finally {
-          assert(
-            streamStatus.get(streamName) === readStatus,
-            X`can't read stream ${q(streamName)}, it's been closed`,
-          );
-          closeStream(streamName);
-          assert(itemCount === 0, X`leftover item count ${q(itemCount)}`);
-        }
-      }
-      return reader();
-    }
-  }
-
-  /**
-   * Write to a stream.
-   *
-   * @param {string} streamName  The stream to be written
-   * @param {string} item  The item to write
-   * @param {Object} position  The position to write the item
-   *
-   * @returns {Object} the new position after writing
-   */
-  function writeStreamItem(streamName, item, position) {
-    insistStreamName(streamName);
-    insistStreamPosition(position);
-
-    let fd = streamFds.get(streamName);
-    if (!fd) {
-      const filePath = `${dirPath}/streams/${streamName}.sss`;
-      const mode = fs.existsSync(filePath) ? 'r+' : 'w';
-      fd = fs.openSync(filePath, mode);
-      streamFds.set(streamName, fd);
-      streamStatus.set(streamName, 'write');
-    } else {
-      const status = streamStatus.get(streamName);
-      if (!status) {
-        streamStatus.set(streamName, 'write');
-      } else {
-        assert(
-          status === 'write',
-          X`can't write stream ${q(streamName)} because it's already in use`,
-        );
-      }
-    }
-    activeStreamFds.add(fd);
-
-    const buf = encoder.encode(`${item}\n`);
-    fs.writeSync(fd, buf, 0, buf.length, position.offset);
-    return harden({
-      offset: position.offset + buf.length,
-      itemCount: position.itemCount + 1,
-    });
-  }
-
-  const streamStore = harden({
-    readStream,
-    writeStreamItem,
-    closeStream,
-    STREAM_START,
-  });
+  const streamStore = sqlStreamStore(dirPath, { sqlite3 });
```

Comment:
It probably makes sense to retain the fs-based stream stuff as a separate thing rather than deleting it.  Assuming we can sort out the file descriptor issues, having an option to put the transcripts into simple text files has a lot of debugging utility.